src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -4939,6 +4939,18 @@
      * to the interface literal signifying the definition of an annotation
      * declaration.
      *
+     * <p>For example:</p>
+     * <pre> @Override </pre>
```

Comment:
we need to show structure in real life context, make it: 
```
@Deprecated
private int value; 
```

Reply:
Done.